nsa540_get_info.py

```diff
@@ -32,6 +39,6 @@
         'Content-Type': "application/x-www-form-urlencoded"
         }
     payload_data = "perform=logout"
-    response_logout = ses.post('http://'+str(sys.argv[1])+'/r51209,/adv,/cgi-bin/setuser.cgi', data=payload_data, headers=headers)

```

Comment:
we sould have this as an argument r51209 / r51266

Reply:
good call. I think it's firmware based but not sure.